MVC/controllers.js

```diff
@@ -84,4 +84,17 @@ const patchArticleById = function (req, res, next) {
 
 }
 
-module.exports = { getTopics, getApiEndPoints, getArticleById, getAllArticles, getCommentsByArticleId, postCommentById, patchArticleById }
\ No newline at end of file
+const deleteCommentById = function (req, res, next){
+
+    const {comment_id} = req.params
+    console.log(comment_id)
+    return deleteComment(comment_id).then(({rows})=>{
+        if (rows.length === 0) { return Promise.reject({ status: 404, message: "The id provided does not exist!" }) }
+        res.send(204)
```

Comment:
its our status(204) we want to send back here, the .send method is for sending back a body.